### PR TITLE
Add Heroku

### PR DESCRIPTION
**Pull request by proteusvacuum**
@amstone326

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -11,21 +14,32 @@ class ConfigException(Exception):
...
 
 
 def get_credentials():
     if GITHUB_USER == '' or GITHUB_PW == '':
         return None
     else:
         return GITHUB_USER, GITHUB_PW
+
+
+if not any([REQUIRED_LABELS_ANY, REQUIRED_LABELS_ALL, BANNED_LABELS]):
```
Comment (amstone326) [inline config.py]: Should this also check for the github credentials?

Comment (proteusvacuum) [reply config.py]: Aren't those optional?

Comment (amstone326) [reply config.py]: No, without credentials the status push will fail (401)

File: custom.conf.template [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [Labels]
 # For each parameter, either provide a list of comma-separated labels, or leave blank if not using
-required-labels-any:
+required-labels-any:sample-required-label
```
Comment (amstone326) [inline custom.conf.template]: what's this for?

File: config.py (hunk 1 above)
Comment (proteusvacuum) [reply config.py]: ah. will update.

File: custom.conf.template (hunk 2 above)
Comment (proteusvacuum) [reply custom.conf.template]: we now check on boot if there are any label set at all. Without this, the tests fail. I suppose I could set up a test conf and use that in the tests instead?

Comment (amstone326) [reply custom.conf.template]: Ah gotcha. My hesitation is just atht this is the public-facing template, so it looks a little funny. If it's easier to have something here, I might prefer going all-out in terms of an example, and doing something like:

```
required-labels-any: one-of-these,is,required
required-labels-all: all-of-these,are,required
banned-labels: these,are,banned
```
thoughts?

File: config.py [hunk 3]
```diff
@@ -11,21 +15,39 @@ class ConfigException(Exception):
...
         return GITHUB_USER, GITHUB_PW
+
+
+UNIT_TESTING = any([arg for arg in sys.argv if 'test' in arg])
+
+
+if not UNIT_TESTING:
+    labels_configured = any([REQUIRED_LABELS_ANY, REQUIRED_LABELS_ALL, BANNED_LABELS])
+    credentials_configured = all([GITHUB_PW, GITHUB_USER])
+    if not labels_configured and not credentials_configured:
```
Comment (amstone326) [inline config.py]: shouldn't this be `or` instead of `and` ?

Comment (amstone326) [reply config.py]: @proteusvacuum Hey just flagging that I'm waiting on this last thing to merge this

Comment (proteusvacuum) [reply config.py]: oops, you are right.

**Pull request merged**